dash_bio/package-info.json

```diff
@@ -18,14 +18,10 @@
     "lint": "eslint src --fix",
     "test": "npm run lint:test && npm run format:test && npm run test-unit",
     "test-unit": "jest",
-    "build:js-dev": "webpack --mode development",
     "build:js": "npm run lint && npm run format && webpack --mode production",
-    "build:py": "node ./extract-meta src/lib/components > dash_bio/metadata.json && python get_version_info.py && python -c \"import dash; dash.development.component_loader.generate_classes('dash_bio', 'dash_bio/metadata.json')\"",
-    "build:r": "dash-generate-components ./src/lib/components dash_bio -p package-info.json --r-prefix='dashbio' --r-imports='dash' --r-depends='manhattanly'",
-    "build": "npm run build:js && npm run build:py",
-    "build:all": "npm run build && npm run build:r",
-    "postbuild": "es-check es5 dash_bio/*.js",
-    "build:dev": "npm run build:js-dev && npm run build:py"
+    "build:backends": "node ./extract-meta src/lib/components > dash_bio/metadata.json && python get_version_info.py && dash-generate-components ./src/lib/components dash_bio -p package-info.json --r-prefix='dashbio' --r-imports='dash' --r-depends='manhattanly'",
+    "build": "npm run build:js && npm run build:backends",
+    "postbuild": "es-check es5 dash_bio/*.js"
   },
   "author": "The Plotly Team <dashbio@plotly.com>",
   "maintainer": "Ryan Patrick Kyle <ryan@plotly.com>",
```

Comment:
We should probably change this... yourself?

Reply:
Yep, changed in 4e6cf1d.